Gemfile.lock

```diff
@@ -34,13 +34,6 @@ GIT
     strip_attributes (1.8.0)
       activemodel (>= 3.0, < 6.0)
 
-GIT
```

Comment:
@sagepe also need these changes in `Gemfile.rails_next.lock` which can be done using `bin/bundle_next`

Reply:
Done